pkg/syncer/cnsoperator/types/types.go

```diff
@@ -41,4 +43,6 @@ const (
 	LabelVirtualMachineName = "vm.consumer.storage.com/name"
 	// Label that points to a StoragePolicyReservation CR name. This is added to CNSRegisterVolume CR and PVC.
 	LabelStoragePolicyReservationName = "quota.storage.vmware.com/storagepolicyreservation-name"
+	// MaxBackOffDurationForReconciler for supervisor APIs is set to 5 minutes
+	MaxBackOffDurationForReconciler = 5 * time.Minute
```

Comment:
This should probably be configurable from driver args with a default of 5 mins. what do you think?

Reply:
will generate another PR to read max back off timeout from config secret